routes/articles.js

```diff
@@ -10,4 +10,15 @@ router.get('/', (req, res) => {
     })
 })
 
+router.post('/article', (req,res) => {
+    const db =req.app.get('db')
+    const title = req.body.title
+    const body = req.body.body
+    const author = req.body.author
+    db.query('insert into articles(??,??,??)',[title,body,author], (err) => {
+        if(err) console.log('408')
+        res('200')
```

Comment:
This won't work. You need to set the response status and end the connection:
```
res.status(200)
res.end()
```
The reason we don't call `res.end()` when we have `res.json(...)` or `res.send(...)` is because these functions end the connection automatically.